ci/lava/qcs6490-rb3gen2-core-kit/boot.yaml

```diff
@@ -53,6 +53,12 @@ actions:
       parameters:
         SKIP_INSTALL: "True"
         TESTS: "pwd, uname -a, ip a"
+    - from: git
+      name: "wlan-smoke-test"
+      path: automated/linux/wlan-smoke/wlan-smoke.yaml
+      repository: https://github.com/linaro/test-definitions.git
+      parameters:
```

Comment:
Does the wifi device get enumerated as 'wlan0' ?
If not I believe we would need to override it with the DEVICE parameter here